packages/driver/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "ES2016",
```

Comment:
doesn't do anything since transpilation target is handled by babel in webpack.config.base.ts, but why not change it here